packages/tpetra/core/test/MatrixMatrix/MatrixMatrix_UnitTests.cpp

```diff
@@ -856,7 +855,8 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL(Tpetra_MatMat, operations_test,SC,LO, GO, NT)
     if(numProcs > 0 && comm->getSize() != numProcs) continue;
 
 
-    double epsilon = currentSystem.get<double> ("epsilon", defaultEpsilon);
+    double epsilon = currentSystem.get<double> ("epsilon",
```

Comment:
Would be nicer if these were the magnitude type based on SC, instead of double.  That would make this work with `__float128` etc.